src/collective/volto/formsupport/scripts/send_exported_forms.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+from AccessControl.SecurityManagement import newSecurityManager
+from collective.volto.formsupport import _
+from datetime import datetime
+from email.mime.application import MIMEApplication
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from io import StringIO
+from logging import getLogger
+from plone import api
+from plone.registry.interfaces import IRegistry
+from plone.volto.interfaces import IVoltoSettings
+from Products.CMFCore.interfaces import IContentish
+from zope.annotation.interfaces import IAnnotations
+from zope.component import getUtility
+from zope.component import queryMultiAdapter
+from zope.globalrequest import getRequest
+
+import click
+import csv
+import sys
+import transaction
+
+
+LAST_SENDING_DATE_KEY = "collvective.volto.formsupport.LAST_SENDING_DATE"
+
+logger = getLogger(__name__)
+
+
+@click.command(
+    help="bin/instance -OPlone run bin/formsupport_data_cleansing [--dryrun|--no-dryrun]",
+    context_settings=dict(
+        ignore_unknown_options=True,
+        allow_extra_args=True,
+    ),
+)
+@click.option(
+    "--dryrun/--no-dryrun",
+    is_flag=True,
+    default=True,
+    help="--dryrun (default) simulate, --no-dryrun actually save the changes",
+)
+def main(dryrun):
+    def send_csv_email(context, csv_file, recipients):
+        host = api.portal.get_tool("MailHost")
+        portal_state = api.content.get_view(
+            name="plone_portal_state",
+            context=api.portal.get(),
+            request=getRequest(),
+        )
+        portal_url = portal_state.navigation_root_url()
+        registry = getUtility(IRegistry)
+        settings = registry.forInterface(IVoltoSettings, prefix="volto", check=False)
+        settings_frontend_domain = getattr(settings, "frontend_domain", None)
+        if (
+            settings_frontend_domain
+            and settings_frontend_domain != "http://localhost:3000"
+        ):
+            portal_url = settings_frontend_domain
+        if portal_url.endswith("/"):
+            portal_url = portal_url[:-1]
+
+        mfrom = api.portal.get_registry_record("plone.email_from_address")
+        mail_attachment_filename = (
+            "FORMS_"
+            + context.Title()
+            + "_"
+            + datetime.today().strftime("%Y-%m-%d")
+            + ".csv"
+        )
+
+        csv_file.seek(0)
+
+        mail_attachment = MIMEApplication(
+            csv_file.read(),
+            Name=mail_attachment_filename,
+        )
+        mail_attachment["Content-Type"] = "text/csv"
+        mail_attachment["Content-Disposition"] = (
+            'attachment; filename="%s"' % mail_attachment_filename
+        )
+
+        for recipient in recipients:
+            if not recipient:
+                continue
+            msg = MIMEMultipart()
+            msg.attach(
+                MIMEText(
+                    api.portal.translate(
```

Comment:
Do you have checked if translations really works? We are in a script, don't take it for granted.

Reply:
works, tested